### ISSUE TITLE
Wrong value calculated between scrollbar selection

### ISSUE THREAD

**Issue by rammussarov** [amcharts/amcharts4#830]
I have time based line chart with one series. Date axis (X) and value axis (Y).

The chart has legend
```
series.legendSettings.labelText = "[bold]Sum:[/]";
series.legendSettings.valueText = "{valueY.sum}";
```

and scrollbar

```
chart.scrollbarX = new am4charts.XYChartScrollbar();
chart.scrollbarX.series.push(series);
```

When date range is changed wrong sum calculated, for example: 
![sum bug](https://user-images.githubusercontent.com/23136499/51248647-04b41400-19bb-11e9-9f92-90634d35b8e4.jpg)

The range between 25th and 28th January chosen.
Jan 25: 80, Jan 26: 87, Jan 27: 84, Jan 28: 83.

The sum is 384 but 584 is shown.

Link to JSFiddle: [click](http://jsfiddle.net/trL52a98/)

**Comment by tvb** [comment]
I'm seeing the same issue

**Comment by martynasma** [comment]
Fixed in [4.9.27](https://github.com/amcharts/amcharts4/releases/tag/4.9.27).

## [4.9.27] - 2020-06-18

### Added
- `ExportMenu` item definition can now have `title` property. If set, it will be added to item's `title` property, which should show a system tooltip on hover.

### Changed
- Hex color parser now supports RGBA format which includes alpha value, e.g. `"#FF000080"` will produce a 50% transparent red.

### Fixed
- Fixed performance issue with a logarithmic `ValueAxis` with `strictMinMax` set to `true`.
- Calculated value of `sum` in the selection of an `XYChart` could include one extra data item, not visible in the selection.
- Pie slices with 0 values were sorted incorrectly in FF.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
